webrtc-stats.html

```diff
@@ -1066,6 +1078,27 @@
                   section 6.4.1.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>lastPacketSentTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the timestamp at which the last packet was sent for this media stream track.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>averageRTCPInterval</code></dfn> of type <span class=
```

Comment:
We currently do not have timestamps nor counters on when the RTCP reports were sent or received. Although a cleaner approach, I am not convinced if those counters brings value.

IMO, the variation in the RTCP interval is not interesting, so the average as is works for me. Having it available as a single metric vs calculating by three metrics, I am on the fence on this, If there would be more use-cases for those additional metrics, I'd be more than happy to change my position on this.